lms/static/js/dashboard/views/dashboard_course_list_factory.js

```diff
@@ -0,0 +1,14 @@
+;(function (define) {
```

Comment:
@clintonb I followed this as an example [dashboard_search_factory.js#L1](https://github.com/edx/edx-platform/blob/master/lms/static/js/search/dashboard/dashboard_search_factory.js#L1). Not sure what is expected here